blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineRunImpl.java

```diff
@@ -95,7 +96,7 @@ public PullRequest getPullRequest() {
                     cnt++;
                     String id = e.getCommitId();
                     if (id == null) id = String.valueOf(cnt);
-                    m.put(id, new ChangeSetResource(e, this));
+                    m.put(id, new ChangeSetResource(OrganizationUtil.getOrganization(this.getOrganization(), true), e, this));
```

Comment:
You are right, this had to be this way in 1.1 branch, not for master. Changing it.